src/main/java/uk/gov/hmcts/reform/pip/rules/repository/InMemoryRepository.java

```diff
@@ -20,6 +36,25 @@
 
     private static final Integer AVAILABLE_PUBLICATION = 1;
 
+
+
+    public InMemoryRepository() throws IOException {
+
+        ObjectMapper om = new ObjectMapper();
+        DateFormat df = new SimpleDateFormat("dd/MM/yyyy");
+        om.setDateFormat(df);
```

Comment:
can you merge the above line into these params

Reply:
done